slatedb-go/go/db_reader_test.go

```diff
@@ -23,7 +23,7 @@ var _ = Describe("DbReader", func() {
 		envFile, err := createEnvFile(tmpDir)
 		Expect(err).NotTo(HaveOccurred())
 
-		db, err := slatedb.Open(tmpDir, slatedb.WithEnvFile[slatedb.DbConfig](envFile))
+		db, err = slatedb.Open(tmpDir, slatedb.WithEnvFile[slatedb.DbConfig](envFile))
```

Comment:
I don't think we should be making tests stateful. Right now, we tear down the DB between each test. This is a good practice. I don't want to maintain state between tests.

Reply:
I’m deleting database after each test in AfterEach